README.md

```diff
@@ -296,6 +296,21 @@ local directory **ls_tmp** must contains the three files (server.test.pem, serve
 
 You can point your `aws` CLI to use the local infrastructure, for example:
 
+Use the below command to install `aws CLI`, if not installed already.
```

Comment:
Sorry, I hate to be nitpicking here again, but can we please make sure that the sequence of paragraphs in the README is still consistent after we apply this change?

Currently, the paragraph above contains this:
```
 You can point your `aws` CLI to use the local infrastructure, for example:
```

... and the paragraph below your change contains this:
```
aws --endpoint-url=http://localhost:4568 kinesis list-streams
{
    "StreamNames": []
}
```
... (although these parts belong together and shouldn't be divided into separate subsections).

Overall, I would suggest something like this:
```
## Accessing the infrastructure via CLI or code

You can point your `aws` CLI to use the local infrastructure, for example:
---
aws --endpoint-url=http://localhost:4568 kinesis list-streams
{
    "StreamNames": []
}
---

Use the below command to install `aws CLI`, if not installed already.
---
pip install awscli
---

### Setting up local region and credentials to run requests against LocaLstack

aws requires the region and the credentials to be set in order to run the aws commands. Create the default configuration & the credentials. Below key will ask for the Access key id, secret Access Key, region & output format.
---
aws configure --profile default
# Config & credential file will be created under ~/.aws folder
---

**NEW**: Check out [awslocal](https://github.com/localstack/awscli-local), a thin CLI wrapper ...
```

Reply:
:+1: